### PR TITLE
add lr scheduler

### PR DESCRIPTION
**Pull request by b-biswas**
adding the option to reduce learning rate as training progresses

### CODE REVIEW

**Comment by b-biswas** [comment]
@aboucaud, I merged this PR because I need to train the model but let me know if you have any comments. 
I will do the modification in another PR.

File: debvader/train.py [hunk 1]
```diff
@@ -87,6 +93,18 @@ def define_callbacks(vae_or_deblender, survey_name, weights_save_path=None):
...
+
+        def scheduler(epoch, lr):
+            if epoch % lr_scheduler_epochs != 0:
+                return lr
+            else:
+                return lr * tf.math.exp(-0.1)
+
+        lr_scheduler = tf.keras.callbacks.LearningRateScheduler(scheduler)
+
+        callbacks = callbacks + [lr_scheduler]
```
Comment (aboucaud) [inline debvader/train.py:106]: use `+=` rather than `var = var +`
